.gitignore

```diff
@@ -125,3 +125,6 @@ e2e/remotetasks/input/ecs.vars
 
 # local terraform overrides
 *.auto.tfvars
+
+# vscode config
+.vscode/launch.json
```

Comment:
Mhm people can add that to their ~/.gitignore and have it apply locally instead of adding arbritrary ignores for every ide/editor out there?

Reply:
We have other commonly used editor files in the .gitignore. It reduces the amount of churn, particularly from external contributors. I'd leave it in. (But maybe move it up so that it's group with `./idea`, `.iml`, etc.)